resources/logging/charts/fluentbit/templates/daemonset.yaml

```diff
@@ -82,6 +82,19 @@ spec:
           mountPath: /secure/forward-tls-ca.crt
           subPath: forward-tls-ca.crt
 {{- end }}
+{{- if and (.Values.backend.http.tls_cert) (.Values.backend.http.tls_key) }}
+        - name: http-tls-secret
```

Comment:
You need to add the two secrets to the secrets.yaml, at the moment that mount will fail